keyboards/macrocat/keymaps/default/keymap.c

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 catmunch (@catmunch)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = {
+        {KC_PPLS,   KC_9,       KC_8,   KC_7 },
+        {KC_PMNS,   KC_6,       KC_5,   KC_4 },
+        {KC_PDOT,   KC_3,       KC_2,   KC_1 },
+        {KC_PENT,   KC_SPACE,   KC_P0,  MO(1)}
+    },
+    [1] = {
+        {KC_PAST,   KC_NO,      KC_NO,  KC_NO},
+        {KC_PSLS,   KC_LPRN,    KC_UP,  KC_RPRN},
+        {KC_COMM,   KC_LEFT,    KC_DOWN,KC_RIGHT},
+        {KC_TAB,    KC_BSPC,    KC_P0,  KC_TRNS}
+    },
+    [2] = {
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO},
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO},
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO},
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO}
+    },
+    [3] = {
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO},
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO},
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO},
+        {KC_NO,     KC_NO,      KC_NO,  KC_NO}
+    }
```

Comment:
```suggestion
    }
```

Reply:
Thanks for your review, but in our design these two layers are for letting users put their own macros. Deleting these will cause some storage problems of layer 2&3 in VIA.

Reply:
This is not a concern of a default keymap, there should be a via keymap.

Reply:
Okay, I created a separate via keymap.